extensions/interactions/TextInput/webdriverio.js

```diff
@@ -66,6 +66,7 @@ var submitAnswer = async function(elem, answer) {
   if (await textAreaElem.isExisting()) {
     await action.setValue('Text Area Input', textAreaElem, answer);
     var submitAnswerBtn = $('.e2e-test-submit-answer-button');
+    await submitAnswerBtn.scrollIntoView();
```

Comment:
Why was this added? Ditto elsewhere in this file.

Reply:
submitAnswerButton hides behind footer when we practice questions for skills. So clicking on answer button was giving error: `Element is not clickable at point (x, x). Other element would receive the click:`. By scolling button into the view before clicking was resolving this error. Also this error only occurs locally and not on Cl.